usds/justice40-tool
# Fix typos found in en.json files and copy

# Purpose
- closes #1629 

# QA
@katherinedm-usds - thanks for this exhaustive list of changes. It made checking and creating this QA spec very easy (it was just copy paste!) 

## QA links

QA link for changes that are on the site are [here](http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/1648-f741d3/en/#3/33.47/-97.5).
QA link for changes that are just in the en.json file and not on the site is [here](https://github.com/usds/justice40-tool/blob/06d07da36dd311c2b409d0f7f5b2902b72eb818d/client/src/intl/en.json).

## QA spec

There are typos in the en.json that should be addressed

- [ ] "disavantaged" should be changed to "disadvantaged", this is viewable on the site
`    "defaultMessage": "Use the map to see communities that are identified as disadvantaged. The map uses publicly-available, nationally-consistent datasets. Learn more about the methodology and datasets that were used to identify disavantaged communities in the current version of the tool on the <link1>Methodology & data</link1> page.",`

![image.png](https://images.zenhubusercontent.com/609ab603576fa56647137b9e/48e15a96-d2a8-432f-b862-0a14a05171e3)

- [ ] "calender" should be "calendar" - this is visible on the footer
```
  "common.pages.footer.eng.cal.text": {
    "defaultMessage": "Engagement calender",
```
![image.png](https://images.zenhubusercontent.com/609ab603576fa56647137b9e/e0cd0c66-81e9-451c-9000-a8779160686f)

- [ ] "infomation" should be changed to "information", this is visible on the footer
```
  "common.pages.footer.rfi.text": {
    "defaultMessage": "Request for Infomation",
```

`![image.png](https://images.zenhubusercontent.com/609ab603576fa56647137b9e/de21e537-fe8c-4107-b54f-6f56bb67b58f)`

- [ ] "catetory" should be "category" - this is visible on the site
```
  "methodology.page.datasets.all.except.workforce.used.in.text": {
    "defaultMessage": "All categories except for the training and workforce development catetory",
```
![image.png](https://images.zenhubusercontent.com/609ab603576fa56647137b9e/3636bf23-2489-47c5-8e01-4b6b3b353748)

- [ ] "Consulation" should be replaced with "Consultation", this is visible on the site

`and Executive Order 13175 on <link4>Consulation and Coordination With Indian Tribal Governments</link4>.",`

![image.png](https://images.zenhubusercontent.com/609ab603576fa56647137b9e/d6e4603b-28d5-4bd9-a668-4a1aa313b0e0)

- [ ] "wasterwater" should be replaced with "wastewater", this is viewable on the site but also should be replaced in all places it occurs on the en.json
```
  "indicator.categories.clean.water.title": {
    "defaultMessage": "Critical clean water and wasterwater infrastructure",
    "description": "Navigate to the methodology page. Navigate to the category section. This will set the category title"
  },
```
![image.png](https://images.zenhubusercontent.com/609ab603576fa56647137b9e/b02abf3f-3c3d-4cbe-981a-c17b49755cee)

```
    "description": "Navigate to the methodology page. Navigate to the dataset section. This is the portion of the dataset card that populates the Used in section for the Critical clean water and wasterwater infrastructure methodology"
  },
```


- [ ] "oridnal" should be "ordinal" - this is not visible on the site, but still should be fixed
```
    "description": "Navigate to the explore the tool page. Click on the map. The side panel will show categories. Open a category. This will define the indicator value's oridinal suffix. For example the st in 91st, the rd in 23rd, and the th in 26th, etc."
  },
```

 - [ ] "Somoa" should be "Samoa", this is not visible but should still be fixed  in all places it occurs
```
"defaultMessage": "American Samoa",
    "description": "On the explore the tool page, on the map, the full name indicating the bounds of American Somoa"
  },

```
- [ ] "desciption" should be changed to "description", this comes up many times and though it's not visible on the site, it should be replaced in all cases
```
  "explore.tool.page.side.panel.indicator.description.asthma": {
    "defaultMessage": "Weighted percent of people who have been told they have asthma",
    "description": "Navigate to the explore the tool page. When the map is in view, click on the map. The side panel will show an indicator desciption of Number of people who have been told they have asthma"
  },
```

- [ ] "naturhazards" should be changed to "natural hazards", this is not viewable on the site but should be changed
```
    "description": "Navigate to the explore the tool page. When the map is in view, click on the map. The side panel will show an indicator desciption of Economic loss rate to agriculture resulting from naturhazards\n    "
  },
```

- [ ] "Meidan" should be changed to "Median", it's not visible on the site but should be changed

```
  "description": "Navigate to the explore the tool page. When the map is in view, click on the map. The side panel will show an indicator desciption of Meidan home value in area"
  },
```


- [ ] "mimicing" should be changed to "mimicking", this is not viewable on the site but still should be updated

```
  "explore.tool.page.side.panel.info.alt.text.icon1": {
    "defaultMessage": "An icon that has depicts pieces of a block selected mimicing the census block census tracts",
```


- [ ] "panelwill" should be updated to "panel will", this is not viewable on the site but still should be changed

```
    "description": "Navigate to the explore the tool page. When the map is in view, click on the map. The side panelwill show Things to know, this is the fifth paragraph of this side pane"
  },
```


- [ ] "Methodoloy" should be replaced with "Methodology", this is not viewable on the site but should be replaced in all places it appears
```
    "defaultMessage": "American Samoa and the Northern Mariana Islands",
    "description": "Methodoloy page dataset card available for American Samoa and Northern Mariana Islands type"
  },
```

- [ ] "engagment" should be changed to "engagement", this is not viewable on the site
```
    "description": "Navigate to the the public engagement page, this will be the public engagment page event registration link label"
  },
```

- [ ] any instance of "the the" replace with "the", this is not viewable on the site

```
    "defaultMessage": "Watch a recorded version of the CEJST training on YouTube.",
    "description": "Navigate to the the public engagement page, there will be box that allows users to watch previously recorded videos. This is that box body text."
  },
```
